systemtest/jpsreport_tests/method_A/test2/02_method_A.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+#---------
+# one pedestrian created at frame 0
```

Comment:
Are you comparing two measuring points? The old test compared two time series. What does the result tell us?

Reply:
Solved due to various explanations in this PR